<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Инспектор аэропортов</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="1" colspan="2">
     <widget class="QPushButton" name="pb_showStatistics">
      <property name="text">
       <string>Показать загруженность</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QGroupBox" name="gb_dateZone">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="l_date">
         <property name="text">
          <string>Дата</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="de_flightDate"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0" colspan="3">
     <widget class="QLabel" name="l_status">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1" rowspan="2" colspan="2">
     <widget class="QPushButton" name="pb_flyList">
      <property name="text">
       <string>Показать список рейсов</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" rowspan="3" colspan="2">
     <widget class="QGroupBox" name="gb_airportZone">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="l_airportName">
         <property name="text">
          <string>Аэропорты</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cb_airports"/>
       </item>
       <item>
        <widget class="QGroupBox" name="gb_arrivalZone">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="rb_departures">
            <property name="text">
             <string>Прилет</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_arrival">
            <property name="text">
             <string>Вылет</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" rowspan="7">
     <widget class="QTableWidget" name="tw_main"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Инспектор аэропортов</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
